core/controllers/contributor_dashboard.py

```diff
@@ -362,7 +362,7 @@ def _get_reviewable_exploration_opportunity_summaries(
         # 3. Get the reviewable translation suggestion target IDs for the user.
         # 4. Get story exploration nodes in order, filtering for explorations
         # that have in review translation suggestions.
-        if topic_name is None:
+        if topic_name is None or topic_name == 'All':
```

Comment:
Initially, to fix this error: 
<img width="1031" alt="Screenshot 2023-04-04 at 9 12 46 AM" src="https://user-images.githubusercontent.com/2841740/229853363-c4513de8-bfc8-49c3-aef6-39095642772d.png">
But now that you mention https://github.com/oppia/oppia/issues/15710, then keeping the backend supporting 'All' makes even more sense now as it would need to be added in the for #15710 anyways.